circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```

Comment:
Yep, [Circle CI uses NVM](https://github.com/circleci/circleci-docs/blob/master/jekyll/_docs/configuration.md#nodejs-version), change your `5.3.0` to `6` and you'll always have the latest version of the 6.x.x branch, stylelint also tests against both the [4.x LTS and 6.x Current](https://github.com/stylelint/stylelint/blob/master/.travis.yml#L5) branches FWIW: